lib/facility.rb

```diff
@@ -1,14 +1,79 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, 
```

Comment:
attr_accessor for these attributes would be preferred - this gives you read and write functionality! No need to have both for all the attributes